src/components/ranking-chart/ranking-chart-styles.module.scss

```diff
@@ -21,8 +21,7 @@
 }
 
 .header {
-  width: 100%;
-  display: flex;
+  width: 80%;
```

Comment:
This is messing a bit with the position of the header text on different views. It was staying stable before. We might need to review it:
Really wide screens
![image](https://user-images.githubusercontent.com/9701591/95962469-b12c5f80-0e06-11eb-8b67-8fe225349ba0.png)
1115px
![image](https://user-images.githubusercontent.com/9701591/95962705-05374400-0e07-11eb-9bd6-62dd6ba045cd.png)